### ISSUE TITLE
Juggling Async: Solution doesn't work until "proper" solution tried

### ISSUE THREAD

**Issue by decani** [rvagg/learnyounode#209]
Similar to issue #170, my solution didn't work until I tried the official solution at https://github.com/rvagg/learnyounode/blob/master/exercises/juggling_async/solution/solution.js, after which my solution passed. I was receiving the same error about undefined protocol "c:".

I am running: -
* Node: 0.10.36
* learnyounode: 2.3.3
* Windows 8.1

My solution: -
```
var http = require("http");

var args = process.argv;
var remaining = args.length - 2;
var res = [];

for (var i = 0; i < args.length - 2; i++) {
	var url = args[i + 2];
	
	(function() {
		var j = i;
		myGet(url, function(data) {
			res[j] = data;
			if (--remaining == 0) {
				printAll(res);
			};
		});
	})();
}

function myGet(url, handler) {
	http.get(url, function(response) {
		var res = "";
		
		response.on("data", function(data) {
			res = res + data;
		});
		
		response.on("end", function() {
			res = res.toString();
			handler(res);
		});
	});
}

function printAll(res) {
	for (var i = 0; i < res.length; i++) {
		console.log(res[i]);
	}
}
```

**Comment by rvagg** [comment]
does `learnyounode run solution.js` work properly for this too now?

**Comment by decani** [comment]
Yes, `learnyounode run ex9.js` works (given that my solution is saved as ex9.js).

BTW, until now, I wasn't using `learnyounode run ...`, I was using `node ...`

**Comment by rvagg** [comment]
well, if you run with `node ex9.js` you'll also have to provide URL arguments after that, when you `learnyounode run ex9.js` it's like you've run with `node ex9.js` except that it feeds in some URL parameters as arguments; basically setting up a test environment complete with fixtures in the form of servers on localhost to fetch stuff from.

**Comment by decani** [comment]
Yes, apologies, I did run it with 3 URL's `node ex9.js http://google.com http://www.jobserve.com/ http://news.bbc.co.uk` - and it still failed. 

So, even with the 3x URL's it didn't work until I ran the "proper" solution.
